src/lnd-actions/get-transactions.js

```diff
@@ -0,0 +1,21 @@
+const { deadline } = require('../grpc-utils')
+
+/**
+ * Returns a list of all on-chain transactions for the engine
+ *
+ * @see http://api.lightning.community/#getTransactions
+ * @param {Object} opts
+ * @param {LndClient} opts.client
+ * @returns {Array<Object>} response
+ */
+function getTransactions ({ client }) {
+  return new Promise((resolve, reject) => {
+    client.getTransactions({}, { deadline: deadline() }, (err, res) => {
+      if (err) return reject(err)
+      const { transactions = [] } = res
```

Comment:
do we do these kinds of transformations in other lnd actions? seems kind of risky since this code is largely untested